src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,523 @@
+import COLOURS from '../../util/COLOURS';
+
+const ProductionLocationInfo = ({
+    classes,
+    countriesOptions,
+    fetchCountries,
+    facilityProcessingTypeOptions,
+    fetchFacilityProcessingType,
+    numberOfWorkersOptions,
+    fetchNumberOfWorkers,
+}) => {
+    const location = useLocation();
+
+    const queryParams = new URLSearchParams(location.search);
+    const nameInQuery = queryParams.get('name');
+    const addressInQuery = queryParams.get('address');
+    const countryInQuery = queryParams.get('country');
+    const [isExpanded, setIsExpanded] = useState(false);
+    const [inputName, setInputName] = useState(nameInQuery ?? '');
+    const [inputAddress, setInputAddress] = useState(addressInQuery ?? '');
+    const [inputCountry, setInputCountry] = useState(null);
+    const [nameTouched, setNameTouched] = useState(false);
+    const [addressTouched, setAddressTouched] = useState(false);
+    const [sector, setSector] = useState('');
+    const [productType, setProductType] = useState([]);
+    const [locationType, setLocationType] = useState(null);
+    const [processingType, setProcessingType] = useState(null);
+    const [numberOfWorkers, setNumberOfWorkers] = useState(null);
+    const [parentCompany, setParentCompany] = useState([]);
+
+    const selectStyles = {
+        control: provided => ({
+            ...provided,
+            height: '56px',
+            borderRadius: '0',
+            '&:focus,&:active,&:focus-within': {
+                borderColor: COLOURS.PURPLE,
+                boxShadow: `0 0 0 1px ${COLOURS.PURPLE}`,
+            },
+        }),
+    };
+    const validate = val => {
+        if (val) {
+            return val.length > 0;
+        }
+        return false;
+    };
+    const toggleExpand = () => {
+        setIsExpanded(!isExpanded);
+    };
+    const handleNameChange = event => {
+        setNameTouched(true);
+        setInputName(event.target.value);
+    };
+    const handleAddressChange = event => {
+        setAddressTouched(true);
+        setInputAddress(event.target.value);
+    };
+    const handleCountryChange = event => setInputCountry(event);
+
+    useEffect(() => {
+        if (!countriesOptions) {
+            fetchCountries();
+        }
+    }, [countriesOptions, fetchCountries]);
+
+    useEffect(() => {
+        if (countriesOptions && validate(countryInQuery)) {
+            const prefilledCountry = countriesOptions.filter(
+                el => el.value === countryInQuery,
+            );
+
+            handleCountryChange(prefilledCountry[0]);
+        }
+    }, [countriesOptions]);
+
+    useEffect(() => {
+        if (!facilityProcessingTypeOptions) {
+            fetchFacilityProcessingType();
+        }
+    }, [facilityProcessingTypeOptions, fetchFacilityProcessingType]);
+
+    useEffect(() => {
+        if (!numberOfWorkersOptions) {
+            fetchNumberOfWorkers();
+        }
+    }, [numberOfWorkersOptions, fetchNumberOfWorkers]);
```

Comment:
I think the number of workers should be an input field where a user can enter a valid number or range, like in the claim process.